httpcore/_async/connection_pool.py

```diff
@@ -76,6 +76,7 @@ class AsyncConnectionPool(AsyncHTTPTransport):
     * **keepalive_expiry** - `Optional[float]` - The maximum time to allow
     before closing a keep-alive connection.
     * **http2** - `bool` - Enable HTTP/2 support.
+    * **local_addr** - `Optional[bytes]` - Local address to connect from
```

Comment:
Miniature nitpick. Let's use a full stop to match the other cases. "Local address to connect from." 😃

Reply:
Fixed.